conf/nginx.conf.server.template

```diff
@@ -241,7 +241,7 @@ server {
         proxy_ssl_trusted_certificate ./cacert.pem;
         proxy_ssl_server_name on;
 
-        access_log ${LOG_DIR}/range_cache.log range_cache;
+        ${RANGE_ACCESS_LOG_DIRECTIVE}
```

Comment:
Will be `access_log ${LOG_DIR}/range_cache.log range_cache;` when `ENABLE_ACCESS_LOGS=true`.

Will be `access_log off;` when `ENABLE_ACCESS_LOGS` is not set (default).